.github/actions/run-gradle/action.yml

```diff
@@ -13,13 +13,19 @@ runs:
       with:
         distribution: temurin
         java-version: 17
+    - uses: actions/setup-node@v4
```

Comment:
Will be replaced by renovate with commit hash tag, right?

Reply:
Yep, I was too lazy to do it by hand 😇